gherkin/php/src/StringUtils.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Cucumber\Gherkin;
+
+/**
+ * Keeps common string operations in one place using correct unicode functions
+ * (and normalises naming with other implementations)
+ */
+final class StringUtils
```

Comment:
`Str`?

Reply:
Is that a specific library you're recommending, or a naming suggestion? If it's the latter I've stuck to what it's called in Gherkin Java to make it more comprehensible for people working across languages

Reply:
the latter so 👍